src/hooks/useSubscriptions.ts

```diff
@@ -115,7 +126,7 @@ export function useProposalSubscription(proposalId, appAddress) {
 
       rawProposalRef.current = rawProposal
 
-      const transformedProposal = await transformProposalData(proposal, config)
+      const transformedProposal = await transformProposalData(proposal)
```

Comment:
Why do we skip the `config` object?

Reply:
The function was changed 9 months ago. I have talked with fabri about this.

![Captura de ecrã 2021-12-30, às 20 57 22](https://user-images.githubusercontent.com/13184907/147793752-27281d7c-ef72-456e-a1a4-40dfadffa2a2.png)